cmd/tailscale/cli/web.go

```diff
@@ -94,6 +94,7 @@ func runWeb(ctx context.Context, args []string) error {
 		}
 		return nil
 	}
+	log.Printf("web server running on: %s\n", webArgs.listen)
```

Comment:
don't need `\n` with the `log` package. also, you cool with this printing e.g. "web server listening on :8080" and not a URL or IP?

Reply:
I was ok just printing the port, but I think it does make more sense to also print the IP as per @Xe's suggestion.